README.md

```diff
@@ -36,19 +36,19 @@ mysite/_config/search.yml
 ```yaml
 Heyday\Elastica\ElasticaService: # Example of customising the index config on the elastic search server (completely optional).
   index_config:
-    analysis:
-      analyzer:
-        default :
-          type : custom
-          tokenizer : standard
-          filter :
-            - standard
-            - lowercase
-            - stemming_filter
-      filter:
-        stemming_filter:
-          type: snowball
-          language: English
+    settings:
```

Comment:
Where does "settings" come from? The upgraded `ruflin/elastica` lib?

Reply:
yup that's right - https://github.com/ruflin/Elastica/blob/master/src/Index/Settings.php#L79

Reply:
It's sent by `ruflin/elastica` when creating an index, and required by the lower level elasticsearch library, here:

https://github.com/elastic/elasticsearch-php#create-an-index